src/user-synchronizer/README.md

```diff
@@ -0,0 +1,23 @@
+# user-synchronizer
+Synchronize user information from Microsoft Graph / WinBind to database.
+
+## Get Started
+
+```shell
+pipenv install
+pipenv run start
+```
+
+## Configuration
+
+It uses environment variables to configure the behavior of the service:
```

Comment:
Not sure which is a better option - using environment variables or using a yaml configuration file.

Reply:
Either is OK, I am just following [the 12 factors](https://12factor.net/config)